app/src/main/AndroidManifest.xml

```diff
@@ -58,6 +62,8 @@
       </intent-filter>
     </service>
 
+    <service android:name=".notification.NotificationService" />
```

Comment:
Would be nice to have [direct boot](https://developer.android.com/training/articles/direct-boot.html) support too. Maybe you could also explicitly set this to `exported=false` so it's clearer?